docs/web/docs/reference/Blueprints.md

```diff
@@ -0,0 +1,101 @@
+
+
+
+
+# Blueprints
+
+## Static React Task
+
+
+Tasks launched from static blueprints need
+                a prebuilt javascript bundle containing the task. We suggest building
+                with our provided useMephistoTask hook.  
+
+|dest|type|default|help|choices|required|
+| :--- | :--- | :--- | :--- | :--- | :--- |
+|block_qualification|str|???|Specify the name of a qualification used to soft block workers.|None|False|
+|tips_location|str|/Users/etesam1/Documents/Mephisto/mephisto/scripts/local_db/gh_actions/assets/tips.csv|Path to csv file containing tips|None|False|
```

Comment:
your full local file path is being output and committed to the docs here:

> /Users/etesam1/Documents/Mephisto/mephisto/scripts/local_db/gh_actions/assets/tips.csv

Ideally it just says:

> mephisto/scripts/local_db/gh_actions/assets/tips.csv

I think you can figure out the root folder where the Mephisto dir is located and "subtract that out" from here to get the more relative URL

Reply:
I could do that, my idea was that this script gets ran on push to main, so it would get replaced with the GIthub runner, but maybe I should filter it out